### PR TITLE
PLT-5498 Add ability to delete a message

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
This PR adds the ability to delete a message by adding an post options menu context for ios and android that will serve as a base for the other post options.

TODO: Remove the `(message deleted)` from the post list

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-5498

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Added or updated unit tests (required for all new features)
- [x] Has UI changes
- [x] Includes text changes and localization file updates

#### Device Information
This PR was tested on: IOS 10.2.1 iPhone 6s and Android 6.0.1 Samsung J5

#### Screenshots

options-ios

bottomsheet

### CODE REVIEW

File: app/components/post/post.js [hunk 1]
```diff
@@ -112,24 +146,91 @@ export default class Post extends Component {
...
+            );
+        } else {
+            message = (
+                <Text style={messageStyle}>
+                    {post.message}
+                </Text>
+            );
+        }
+
+        if (this.props.post.message.length) {
```
Comment (hmhealey) [inline app/components/post/post.js]: This will prevent the menu from showing up for posts containing only files which I don't think we want

**Comment by hmhealey** [comment]
Also you need to rebase for Jenkins to pick up the changes that'll fix your tests

File: app/components/post/post.js (hunk 1 above)
Comment (enahum) [inline app/components/post/post.js]: Right I didn't thought of that will fix it to wrap the attachments too

**Comment by enahum** [comment]
@hmhealey @jarredwitt rebased and ready for re-review

**Pull request merged**